packages/web-components/fast-foundation/src/file/file.template.ts

```diff
@@ -0,0 +1,30 @@
+import { html, slotted, ViewTemplate } from "@microsoft/fast-element";
+// import { Button } from "../button";
+import type { ElementDefinitionContext } from "../design-system";
+import type { File } from "./file";
+
+/**
+ * The template for {@link @microsoft/fast-foundation#file}  component
+ * @public
+ */
+export const fileTemplate: (
+    context: ElementDefinitionContext
+) => ViewTemplate<File> = context => {
+    return html`
+        <template>
+            <div class="root" part="root">
+                <span part="file-selector-button">
+                    <slot name="fileSelectorButton" ${slotted(
+                        "fileSelectorButton"
+                    )} @click="${x => x.handleClick()}">
+                </span>
```

Comment:
Perhaps we should use the definition API here?

Reply:
> Perhaps we should use the definition API here?
I think this would be a good candidate for that. It might also be a good route for the file list as aspect as well.